### PR TITLE
Nvidia.Broadcast version 1.3.5

### PR DESCRIPTION
**Pull request by ollyroo**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/80632)

### CODE REVIEW

File: manifests/n/Nvidia/Broadcast/1.3.5/Nvidia.Broadcast.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Nvidia.Broadcast
+PackageVersion: 1.3.5
```
Comment (Trenly) [inline manifests/n/Nvidia/Broadcast/1.3.5/Nvidia.Broadcast.installer.yaml:5]:
```suggestion
PackageVersion: 1.3.5.4
```

File: manifests/n/Nvidia/Broadcast/1.3.5/Nvidia.Broadcast.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Nvidia.Broadcast
+PackageVersion: 1.3.5
```
Comment (Trenly) [inline manifests/n/Nvidia/Broadcast/1.3.5/Nvidia.Broadcast.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.3.5.4
```

File: manifests/n/Nvidia/Broadcast/1.3.5/Nvidia.Broadcast.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: Nvidia.Broadcast
+PackageVersion: 1.3.5
```
Comment (Trenly) [inline manifests/n/Nvidia/Broadcast/1.3.5/Nvidia.Broadcast.yaml:5]:
```suggestion
PackageVersion: 1.3.5.4
```

**Review by Trenly — CHANGES_REQUESTED** [review]
When updating the `PackageVersion`, you will need to update the name of the version folder also. You can do this in GitHub web by pressing <kbd>.</kbd> 

image

**Comment by Trenly** [comment]
This is what is causing the no-executables error:
image

**Comment by ollyroo** [comment]
Hmm seems it has the exit code 3858759680 in the manifest but still it doesnt pass
I dont seem to get it do generate an exit code?

**Comment by yao-msft** [comment]
This package requires specific nvidia display drivers installed so the installation may not pass for every machine. But since we already have a previous version checked in and the new version will quit correctly on no-op, so I'm letting this new version through by directly setting Validation-Completed

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**